src/sean/config.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+from typing import Optional
+import torch
+
+
+@dataclass
+class Config:
+    device_name: str = "cuda"
+    device: torch.device = torch.device("cuda")
+    label_nc: int = 19
+    output_nc: int = 3
+    model_path: Optional[str] = None
+    output_dir: str = "output"
+    load_size: int = 256
+    crop_size: int = 256
+
+    name: str = "default"
+    dataroot: str = "data"
+    model_dir: Optional[str] = None
+    checkpoint_dir: str = ""
+    batch_size: int = 2
+    beta1: float = 0.5
+    beta2: float = 0.999
```

Comment:
pape says beta1 = 0 and beta2 = 0.999 but default params of original implementation like this
https://github.com/ZPdesu/SEAN/blob/04c7536ff3fecd2d1a09c9ae046a1144636033a5/options/train_options.py#L27-L28